template.tpl

```diff
@@ -1215,20 +1487,55 @@ const generateOptions = (integration) => {
   };
 };
 
-const processBasicOrGA4Event = (isGA4Event) => {
-  let options = {};
-  if (isGA4Event) {
-    options = generateOptions("Google Analytics 4 Proxy");
+const processInit = () => {
+  // Init handled upstream
+  data.gtmOnSuccess();
+};
+
+const processTrack = () => {
+  const options = {};
+
+  if (data.commonEventName) {
+    const props = parseSimpleTable(data.commonEventProperties || []);
+    track(data.commonEventName, props, options);
+
+    data.gtmOnSuccess();
+  } else {
+      log("ERROR: Freshpaint Track GTM Template missing eventName");
+      data.gtmOnFailure();
   }
+};
+
+const processIdentify = () => {
+    const options = {};
+
+    let identifier = undefined;
+    if (data.identifyIdentifier) {
+      identifier = data.identifyIdentifier;
+    }
```

Comment:
I'm not sure I understand the `identifyIdentifier`. I originally thought that it would be used to lookup a user identity, but it looks like it just gets passed as the identity (though I may be misunderstanding how this all works). What is intended to be set here and how will that help enrich the data on the FP side?

Reply:
the way identify works is:
* if an identifier is passed (vs. undefined), that sets $user_id for subsequent track requests
* When an identifier has not been passed, we use our anonymous id ($device_id)
* distinct_id is set to $user_id if set, otherwise $device_id
* Any user props that are passed are cumulatively added / updated to what userprops are already present

this is *mostly* described here https://documentation.freshpaint.io/readme/setting-up-identify

Reply:
I'd say I have a rough understanding of how identify works in freshpaint, but I'm less clear on what hardcoding a  template value is aiming to achieve.

Reply:
most ppl will still use the fp tag manager for identify (typically on page load and / or login)

this lets them use GTM if they prefer for that purpose